src/main/java/uk/gov/hmcts/cmc/claimstore/utils/DocumentNameUtils.java

```diff
@@ -112,4 +112,16 @@ public static String buildCoverSheetFileBaseName(String number) {
 
         return format("%s-directions-order-cover-sheet", number);
     }
+
+    public static String buildRequestForJudgementByDeterminationFileBaseName(String caseRef) {
+        requireNonBlank(caseRef);
+
+        return format("%s-ccj-request-determination", caseRef);
+    }
+
+    public static String buildRequestForJudgementByAdmissionFileBaseName(String caseRef) {
```

Comment:
Both of these methods can be combined into one by passing additional parameter of type. The calling code will also be more clean.

Reply:
yes I agree but the reason I didn't do that because the type is called "ADMISSIONS" however the file name uses "ADMISSION" (without the s)

Reply:
Does that really matter to have those file names?

Reply:
the name format is mentioned on the user story so I was following what it is said on there